roles/fips/tasks/main.yml

```diff
@@ -1,17 +1,11 @@
 ---
 - include_tasks: 'el{{ ansible_distribution_major_version }}.yml'
+  when: not ansible_fips
```

Comment:
Is this a built in?

Reply:
yeah, since ansible 2.4 or so: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/fips.py